.circleci/config.yml

```diff
@@ -767,6 +767,44 @@ jobs:
                 name: Notify Slack on failure
                 when: on_fail
         working_directory: ~/kubernetes-monitor
+    lint:
```

Comment:
We use the `circleci` binary to pack the configuration into this file. The CircleCI config is split into several files and then combined into this one as noted in the README: https://github.com/snyk/kubernetes-monitor/tree/staging/.circleci Could you also ensure the changes are added outside of this file and then the config.yml is generated?